ChangeLog.d/fix-pkwrite-buffer-overrun.txt

```diff
@@ -0,0 +1,10 @@
+Security
+   * Fix a buffer overflow in mbedtls_pk_write_pubkey(),
+     mbedtls_pk_write_pubkey_der(), mbedtls_pk_write_pubkey_pem(),
+     mbedtls_pk_write_key_der() and mbedtls_pk_write_key_pem().
```

Comment:
`_pem()` is safe, only the DER versions are impacted.

Reply:
Ah whoops, yes. I'll update that.

Reply:
Are you re-doing everything or doing an interactive rebase?

Reply:
A combination of redoing, partly via a script and cherry-picking.

I can't straightforwardly rebase as it destroys merge commits.